### DISCUSSION TITLE
多表关联更新/删除语法

### DISCUSSION THREAD

**Discussion by raykimurayyz** [apache/doris#12555] [A - General / Q&A] [answered]
使用了mysql的多表关联Update和Delete都提示语法错误
有没有人知道具体的语法是怎么写的？还是说Doris不支持这样的场景呢？

另外，多表关联update是不是也需要 Unique Key 模型？

**Comment by LemonLiTree** [comment]
貌似不支持这样做

**Comment by raykimurayyz** [comment, marked as answer]
用了唯一模型的方式变相解决了需求

**Reply by yuanyuan8983** [reply]
是的，update 语句目前仅支持 Unique Key 模型
